endpoints/openrtb2/amp_auction.go

```diff
@@ -182,9 +182,13 @@ func (deps *endpointDeps) loadRequestJSONForAmp(httpRequest *http.Request) (req
 		return
 	}
 
+	debugParam, ok := httpRequest.URL.Query()["debug"]
+	debug := ok && len(debugParam) > 0 && debugParam[0] == "1"
```

Comment:
This could be extracted into common code to check for boolean presence of a query param. We may also want to treat an empty value (e.g. `?debug`) as true.

Reply:
I think it's ok as is.